From 79dad90df43c96099f7b8e69fa8fd67da1fafd08 Mon Sep 17 00:00:00 2001
From: Yu Chien Peter Lin <peterlin@andestech.com>
Date: Tue, 16 Aug 2022 18:27:24 +0800
Subject: [PATCH] riscv: dts: add andes folder

This patch adds andes folder holding ae350 device tree source, we
can run `make dtbs` to generates all the dtbs.

Signed-off-by: Yu Chien Peter Lin <peterlin@andestech.com>
---
 arch/riscv/boot/dts/Makefile                  |  1 +
 arch/riscv/boot/dts/andes/Makefile            | 56 +++++++++++++++++++
 .../ae350_a27l2_c1_noncoherent_32.dts         |  0
 .../ae350_a27l2_c1_noncoherent_32_d.dts       |  0
 .../ae350_a27l2_c1_noncoherent_32_d_dsp.dts   |  0
 .../ae350_a27l2_c1_noncoherent_32_dsp.dts     |  0
 .../ae350_a45_c1_noncoherent_32.dts           |  0
 .../ae350_a45_c1_noncoherent_32_d.dts         |  0
 .../ae350_a45_c1_noncoherent_32_d_dsp.dts     |  0
 .../ae350_a45_c1_noncoherent_32_dsp.dts       |  0
 .../ae350_ax27l2_c1_noncoherent_64.dts        |  0
 .../ae350_ax27l2_c1_noncoherent_64_d.dts      |  0
 .../ae350_ax27l2_c1_noncoherent_64_d_dsp.dts  |  0
 .../ae350_ax27l2_c1_noncoherent_64_dsp.dts    |  0
 .../ae350_ax45_c1_noncoherent_64.dts          |  0
 .../ae350_ax45_c1_noncoherent_64_d.dts        |  0
 .../ae350_ax45_c1_noncoherent_64_d_dsp.dts    |  0
 .../ae350_ax45_c1_noncoherent_64_dsp.dts      |  0
 .../boot/dts/{ => andes}/ae350_c1_32.dts      |  0
 .../boot/dts/{ => andes}/ae350_c1_32_d.dts    |  0
 .../dts/{ => andes}/ae350_c1_32_d_dsp.dts     |  0
 .../boot/dts/{ => andes}/ae350_c1_32_dsp.dts  |  0
 .../boot/dts/{ => andes}/ae350_c1_64.dts      |  0
 .../boot/dts/{ => andes}/ae350_c1_64_d.dts    |  0
 .../dts/{ => andes}/ae350_c1_64_d_dsp.dts     |  0
 .../boot/dts/{ => andes}/ae350_c1_64_dsp.dts  |  0
 .../{ => andes}/ae350_c1_noncoherent_32.dts   |  0
 .../{ => andes}/ae350_c1_noncoherent_32_d.dts |  0
 .../ae350_c1_noncoherent_32_d_dsp.dts         |  0
 .../ae350_c1_noncoherent_32_dsp.dts           |  0
 .../{ => andes}/ae350_c1_noncoherent_64.dts   |  0
 .../{ => andes}/ae350_c1_noncoherent_64_d.dts |  0
 .../ae350_c1_noncoherent_64_d_dsp.dts         |  0
 .../ae350_c1_noncoherent_64_dsp.dts           |  0
 .../boot/dts/{ => andes}/ae350_c2_32.dts      |  0
 .../boot/dts/{ => andes}/ae350_c2_32_d.dts    |  0
 .../dts/{ => andes}/ae350_c2_32_d_dsp.dts     |  0
 .../boot/dts/{ => andes}/ae350_c2_32_dsp.dts  |  0
 .../boot/dts/{ => andes}/ae350_c2_64.dts      |  0
 .../boot/dts/{ => andes}/ae350_c2_64_d.dts    |  0
 .../dts/{ => andes}/ae350_c2_64_d_dsp.dts     |  0
 .../boot/dts/{ => andes}/ae350_c2_64_dsp.dts  |  0
 .../boot/dts/{ => andes}/ae350_c4_32.dts      |  0
 .../boot/dts/{ => andes}/ae350_c4_32_d.dts    |  0
 .../dts/{ => andes}/ae350_c4_32_d_dsp.dts     |  0
 .../boot/dts/{ => andes}/ae350_c4_32_dsp.dts  |  0
 .../boot/dts/{ => andes}/ae350_c4_64.dts      |  0
 .../boot/dts/{ => andes}/ae350_c4_64_d.dts    |  0
 .../dts/{ => andes}/ae350_c4_64_d_dsp.dts     |  0
 .../boot/dts/{ => andes}/ae350_c4_64_dsp.dts  |  0
 .../riscv/boot/dts/{ => andes}/qemu_c1_32.dts |  0
 .../riscv/boot/dts/{ => andes}/qemu_c1_64.dts |  0
 .../riscv/boot/dts/{ => andes}/qemu_c2_32.dts |  0
 .../riscv/boot/dts/{ => andes}/qemu_c2_64.dts |  0
 .../riscv/boot/dts/{ => andes}/qemu_c4_32.dts |  0
 .../riscv/boot/dts/{ => andes}/qemu_c4_64.dts |  0
 56 files changed, 57 insertions(+)
 create mode 100644 arch/riscv/boot/dts/andes/Makefile
 rename arch/riscv/boot/dts/{ => andes}/ae350_a27l2_c1_noncoherent_32.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_a27l2_c1_noncoherent_32_d.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_a27l2_c1_noncoherent_32_d_dsp.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_a27l2_c1_noncoherent_32_dsp.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_a45_c1_noncoherent_32.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_a45_c1_noncoherent_32_d.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_a45_c1_noncoherent_32_d_dsp.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_a45_c1_noncoherent_32_dsp.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_ax27l2_c1_noncoherent_64.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_ax27l2_c1_noncoherent_64_d.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_ax27l2_c1_noncoherent_64_d_dsp.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_ax27l2_c1_noncoherent_64_dsp.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_ax45_c1_noncoherent_64.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_ax45_c1_noncoherent_64_d.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_ax45_c1_noncoherent_64_d_dsp.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_ax45_c1_noncoherent_64_dsp.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c1_32.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c1_32_d.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c1_32_d_dsp.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c1_32_dsp.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c1_64.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c1_64_d.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c1_64_d_dsp.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c1_64_dsp.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c1_noncoherent_32.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c1_noncoherent_32_d.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c1_noncoherent_32_d_dsp.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c1_noncoherent_32_dsp.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c1_noncoherent_64.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c1_noncoherent_64_d.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c1_noncoherent_64_d_dsp.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c1_noncoherent_64_dsp.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c2_32.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c2_32_d.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c2_32_d_dsp.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c2_32_dsp.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c2_64.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c2_64_d.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c2_64_d_dsp.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c2_64_dsp.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c4_32.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c4_32_d.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c4_32_d_dsp.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c4_32_dsp.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c4_64.dts (100%)
 mode change 100755 => 100644
 rename arch/riscv/boot/dts/{ => andes}/ae350_c4_64_d.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c4_64_d_dsp.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/ae350_c4_64_dsp.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/qemu_c1_32.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/qemu_c1_64.dts (100%)
 mode change 100755 => 100644
 rename arch/riscv/boot/dts/{ => andes}/qemu_c2_32.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/qemu_c2_64.dts (100%)
 mode change 100755 => 100644
 rename arch/riscv/boot/dts/{ => andes}/qemu_c4_32.dts (100%)
 rename arch/riscv/boot/dts/{ => andes}/qemu_c4_64.dts (100%)
 mode change 100755 => 100644

diff --git a/arch/riscv/boot/dts/Makefile b/arch/riscv/boot/dts/Makefile
index dcc3ada78455..019782901b18 100644
--- a/arch/riscv/boot/dts/Makefile
+++ b/arch/riscv/boot/dts/Makefile
@@ -1,2 +1,3 @@
 # SPDX-License-Identifier: GPL-2.0
 subdir-y += sifive
+subdir-y += andes
diff --git a/arch/riscv/boot/dts/andes/Makefile b/arch/riscv/boot/dts/andes/Makefile
new file mode 100644
index 000000000000..b59497f9999b
--- /dev/null
+++ b/arch/riscv/boot/dts/andes/Makefile
@@ -0,0 +1,56 @@
+# SPDX-License-Identifier: GPL-2.0
+dtb-$(CONFIG_PLAT_AE350) += \
+  ae350_a27l2_c1_noncoherent_32.dtb \
+  ae350_a27l2_c1_noncoherent_32_d.dtb \
+  ae350_a27l2_c1_noncoherent_32_d_dsp.dtb \
+  ae350_a27l2_c1_noncoherent_32_dsp.dtb \
+  ae350_a45_c1_noncoherent_32.dtb \
+  ae350_a45_c1_noncoherent_32_d.dtb \
+  ae350_a45_c1_noncoherent_32_d_dsp.dtb \
+  ae350_a45_c1_noncoherent_32_dsp.dtb \
+  ae350_ax27l2_c1_noncoherent_64.dtb \
+  ae350_ax27l2_c1_noncoherent_64_d.dtb \
+  ae350_ax27l2_c1_noncoherent_64_d_dsp.dtb \
+  ae350_ax27l2_c1_noncoherent_64_dsp.dtb \
+  ae350_ax45_c1_noncoherent_64.dtb \
+  ae350_ax45_c1_noncoherent_64_d.dtb \
+  ae350_ax45_c1_noncoherent_64_d_dsp.dtb \
+  ae350_ax45_c1_noncoherent_64_dsp.dtb \
+  ae350_c1_32.dtb \
+  ae350_c1_32_d.dtb \
+  ae350_c1_32_d_dsp.dtb \
+  ae350_c1_32_dsp.dtb \
+  ae350_c1_64.dtb \
+  ae350_c1_64_d.dtb \
+  ae350_c1_64_d_dsp.dtb \
+  ae350_c1_64_dsp.dtb \
+  ae350_c1_noncoherent_32.dtb \
+  ae350_c1_noncoherent_32_d.dtb \
+  ae350_c1_noncoherent_32_d_dsp.dtb \
+  ae350_c1_noncoherent_32_dsp.dtb \
+  ae350_c1_noncoherent_64.dtb \
+  ae350_c1_noncoherent_64_d.dtb \
+  ae350_c1_noncoherent_64_d_dsp.dtb \
+  ae350_c1_noncoherent_64_dsp.dtb \
+  ae350_c2_32.dtb \
+  ae350_c2_32_d.dtb \
+  ae350_c2_32_d_dsp.dtb \
+  ae350_c2_32_dsp.dtb \
+  ae350_c2_64.dtb \
+  ae350_c2_64_d.dtb \
+  ae350_c2_64_d_dsp.dtb \
+  ae350_c2_64_dsp.dtb \
+  ae350_c4_32.dtb \
+  ae350_c4_32_d.dtb \
+  ae350_c4_32_d_dsp.dtb \
+  ae350_c4_32_dsp.dtb \
+  ae350_c4_64.dtb \
+  ae350_c4_64_d.dtb \
+  ae350_c4_64_d_dsp.dtb \
+  ae350_c4_64_dsp.dtb \
+  qemu_c1_32.dtb \
+  qemu_c1_64.dtb \
+  qemu_c2_32.dtb \
+  qemu_c2_64.dtb \
+  qemu_c4_32.dtb \
+  qemu_c4_64.dtb
diff --git a/arch/riscv/boot/dts/ae350_a27l2_c1_noncoherent_32.dts b/arch/riscv/boot/dts/andes/ae350_a27l2_c1_noncoherent_32.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_a27l2_c1_noncoherent_32.dts
rename to arch/riscv/boot/dts/andes/ae350_a27l2_c1_noncoherent_32.dts
diff --git a/arch/riscv/boot/dts/ae350_a27l2_c1_noncoherent_32_d.dts b/arch/riscv/boot/dts/andes/ae350_a27l2_c1_noncoherent_32_d.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_a27l2_c1_noncoherent_32_d.dts
rename to arch/riscv/boot/dts/andes/ae350_a27l2_c1_noncoherent_32_d.dts
diff --git a/arch/riscv/boot/dts/ae350_a27l2_c1_noncoherent_32_d_dsp.dts b/arch/riscv/boot/dts/andes/ae350_a27l2_c1_noncoherent_32_d_dsp.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_a27l2_c1_noncoherent_32_d_dsp.dts
rename to arch/riscv/boot/dts/andes/ae350_a27l2_c1_noncoherent_32_d_dsp.dts
diff --git a/arch/riscv/boot/dts/ae350_a27l2_c1_noncoherent_32_dsp.dts b/arch/riscv/boot/dts/andes/ae350_a27l2_c1_noncoherent_32_dsp.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_a27l2_c1_noncoherent_32_dsp.dts
rename to arch/riscv/boot/dts/andes/ae350_a27l2_c1_noncoherent_32_dsp.dts
diff --git a/arch/riscv/boot/dts/ae350_a45_c1_noncoherent_32.dts b/arch/riscv/boot/dts/andes/ae350_a45_c1_noncoherent_32.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_a45_c1_noncoherent_32.dts
rename to arch/riscv/boot/dts/andes/ae350_a45_c1_noncoherent_32.dts
diff --git a/arch/riscv/boot/dts/ae350_a45_c1_noncoherent_32_d.dts b/arch/riscv/boot/dts/andes/ae350_a45_c1_noncoherent_32_d.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_a45_c1_noncoherent_32_d.dts
rename to arch/riscv/boot/dts/andes/ae350_a45_c1_noncoherent_32_d.dts
diff --git a/arch/riscv/boot/dts/ae350_a45_c1_noncoherent_32_d_dsp.dts b/arch/riscv/boot/dts/andes/ae350_a45_c1_noncoherent_32_d_dsp.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_a45_c1_noncoherent_32_d_dsp.dts
rename to arch/riscv/boot/dts/andes/ae350_a45_c1_noncoherent_32_d_dsp.dts
diff --git a/arch/riscv/boot/dts/ae350_a45_c1_noncoherent_32_dsp.dts b/arch/riscv/boot/dts/andes/ae350_a45_c1_noncoherent_32_dsp.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_a45_c1_noncoherent_32_dsp.dts
rename to arch/riscv/boot/dts/andes/ae350_a45_c1_noncoherent_32_dsp.dts
diff --git a/arch/riscv/boot/dts/ae350_ax27l2_c1_noncoherent_64.dts b/arch/riscv/boot/dts/andes/ae350_ax27l2_c1_noncoherent_64.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_ax27l2_c1_noncoherent_64.dts
rename to arch/riscv/boot/dts/andes/ae350_ax27l2_c1_noncoherent_64.dts
diff --git a/arch/riscv/boot/dts/ae350_ax27l2_c1_noncoherent_64_d.dts b/arch/riscv/boot/dts/andes/ae350_ax27l2_c1_noncoherent_64_d.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_ax27l2_c1_noncoherent_64_d.dts
rename to arch/riscv/boot/dts/andes/ae350_ax27l2_c1_noncoherent_64_d.dts
diff --git a/arch/riscv/boot/dts/ae350_ax27l2_c1_noncoherent_64_d_dsp.dts b/arch/riscv/boot/dts/andes/ae350_ax27l2_c1_noncoherent_64_d_dsp.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_ax27l2_c1_noncoherent_64_d_dsp.dts
rename to arch/riscv/boot/dts/andes/ae350_ax27l2_c1_noncoherent_64_d_dsp.dts
diff --git a/arch/riscv/boot/dts/ae350_ax27l2_c1_noncoherent_64_dsp.dts b/arch/riscv/boot/dts/andes/ae350_ax27l2_c1_noncoherent_64_dsp.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_ax27l2_c1_noncoherent_64_dsp.dts
rename to arch/riscv/boot/dts/andes/ae350_ax27l2_c1_noncoherent_64_dsp.dts
diff --git a/arch/riscv/boot/dts/ae350_ax45_c1_noncoherent_64.dts b/arch/riscv/boot/dts/andes/ae350_ax45_c1_noncoherent_64.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_ax45_c1_noncoherent_64.dts
rename to arch/riscv/boot/dts/andes/ae350_ax45_c1_noncoherent_64.dts
diff --git a/arch/riscv/boot/dts/ae350_ax45_c1_noncoherent_64_d.dts b/arch/riscv/boot/dts/andes/ae350_ax45_c1_noncoherent_64_d.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_ax45_c1_noncoherent_64_d.dts
rename to arch/riscv/boot/dts/andes/ae350_ax45_c1_noncoherent_64_d.dts
diff --git a/arch/riscv/boot/dts/ae350_ax45_c1_noncoherent_64_d_dsp.dts b/arch/riscv/boot/dts/andes/ae350_ax45_c1_noncoherent_64_d_dsp.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_ax45_c1_noncoherent_64_d_dsp.dts
rename to arch/riscv/boot/dts/andes/ae350_ax45_c1_noncoherent_64_d_dsp.dts
diff --git a/arch/riscv/boot/dts/ae350_ax45_c1_noncoherent_64_dsp.dts b/arch/riscv/boot/dts/andes/ae350_ax45_c1_noncoherent_64_dsp.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_ax45_c1_noncoherent_64_dsp.dts
rename to arch/riscv/boot/dts/andes/ae350_ax45_c1_noncoherent_64_dsp.dts
diff --git a/arch/riscv/boot/dts/ae350_c1_32.dts b/arch/riscv/boot/dts/andes/ae350_c1_32.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c1_32.dts
rename to arch/riscv/boot/dts/andes/ae350_c1_32.dts
diff --git a/arch/riscv/boot/dts/ae350_c1_32_d.dts b/arch/riscv/boot/dts/andes/ae350_c1_32_d.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c1_32_d.dts
rename to arch/riscv/boot/dts/andes/ae350_c1_32_d.dts
diff --git a/arch/riscv/boot/dts/ae350_c1_32_d_dsp.dts b/arch/riscv/boot/dts/andes/ae350_c1_32_d_dsp.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c1_32_d_dsp.dts
rename to arch/riscv/boot/dts/andes/ae350_c1_32_d_dsp.dts
diff --git a/arch/riscv/boot/dts/ae350_c1_32_dsp.dts b/arch/riscv/boot/dts/andes/ae350_c1_32_dsp.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c1_32_dsp.dts
rename to arch/riscv/boot/dts/andes/ae350_c1_32_dsp.dts
diff --git a/arch/riscv/boot/dts/ae350_c1_64.dts b/arch/riscv/boot/dts/andes/ae350_c1_64.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c1_64.dts
rename to arch/riscv/boot/dts/andes/ae350_c1_64.dts
diff --git a/arch/riscv/boot/dts/ae350_c1_64_d.dts b/arch/riscv/boot/dts/andes/ae350_c1_64_d.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c1_64_d.dts
rename to arch/riscv/boot/dts/andes/ae350_c1_64_d.dts
diff --git a/arch/riscv/boot/dts/ae350_c1_64_d_dsp.dts b/arch/riscv/boot/dts/andes/ae350_c1_64_d_dsp.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c1_64_d_dsp.dts
rename to arch/riscv/boot/dts/andes/ae350_c1_64_d_dsp.dts
diff --git a/arch/riscv/boot/dts/ae350_c1_64_dsp.dts b/arch/riscv/boot/dts/andes/ae350_c1_64_dsp.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c1_64_dsp.dts
rename to arch/riscv/boot/dts/andes/ae350_c1_64_dsp.dts
diff --git a/arch/riscv/boot/dts/ae350_c1_noncoherent_32.dts b/arch/riscv/boot/dts/andes/ae350_c1_noncoherent_32.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c1_noncoherent_32.dts
rename to arch/riscv/boot/dts/andes/ae350_c1_noncoherent_32.dts
diff --git a/arch/riscv/boot/dts/ae350_c1_noncoherent_32_d.dts b/arch/riscv/boot/dts/andes/ae350_c1_noncoherent_32_d.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c1_noncoherent_32_d.dts
rename to arch/riscv/boot/dts/andes/ae350_c1_noncoherent_32_d.dts
diff --git a/arch/riscv/boot/dts/ae350_c1_noncoherent_32_d_dsp.dts b/arch/riscv/boot/dts/andes/ae350_c1_noncoherent_32_d_dsp.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c1_noncoherent_32_d_dsp.dts
rename to arch/riscv/boot/dts/andes/ae350_c1_noncoherent_32_d_dsp.dts
diff --git a/arch/riscv/boot/dts/ae350_c1_noncoherent_32_dsp.dts b/arch/riscv/boot/dts/andes/ae350_c1_noncoherent_32_dsp.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c1_noncoherent_32_dsp.dts
rename to arch/riscv/boot/dts/andes/ae350_c1_noncoherent_32_dsp.dts
diff --git a/arch/riscv/boot/dts/ae350_c1_noncoherent_64.dts b/arch/riscv/boot/dts/andes/ae350_c1_noncoherent_64.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c1_noncoherent_64.dts
rename to arch/riscv/boot/dts/andes/ae350_c1_noncoherent_64.dts
diff --git a/arch/riscv/boot/dts/ae350_c1_noncoherent_64_d.dts b/arch/riscv/boot/dts/andes/ae350_c1_noncoherent_64_d.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c1_noncoherent_64_d.dts
rename to arch/riscv/boot/dts/andes/ae350_c1_noncoherent_64_d.dts
diff --git a/arch/riscv/boot/dts/ae350_c1_noncoherent_64_d_dsp.dts b/arch/riscv/boot/dts/andes/ae350_c1_noncoherent_64_d_dsp.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c1_noncoherent_64_d_dsp.dts
rename to arch/riscv/boot/dts/andes/ae350_c1_noncoherent_64_d_dsp.dts
diff --git a/arch/riscv/boot/dts/ae350_c1_noncoherent_64_dsp.dts b/arch/riscv/boot/dts/andes/ae350_c1_noncoherent_64_dsp.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c1_noncoherent_64_dsp.dts
rename to arch/riscv/boot/dts/andes/ae350_c1_noncoherent_64_dsp.dts
diff --git a/arch/riscv/boot/dts/ae350_c2_32.dts b/arch/riscv/boot/dts/andes/ae350_c2_32.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c2_32.dts
rename to arch/riscv/boot/dts/andes/ae350_c2_32.dts
diff --git a/arch/riscv/boot/dts/ae350_c2_32_d.dts b/arch/riscv/boot/dts/andes/ae350_c2_32_d.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c2_32_d.dts
rename to arch/riscv/boot/dts/andes/ae350_c2_32_d.dts
diff --git a/arch/riscv/boot/dts/ae350_c2_32_d_dsp.dts b/arch/riscv/boot/dts/andes/ae350_c2_32_d_dsp.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c2_32_d_dsp.dts
rename to arch/riscv/boot/dts/andes/ae350_c2_32_d_dsp.dts
diff --git a/arch/riscv/boot/dts/ae350_c2_32_dsp.dts b/arch/riscv/boot/dts/andes/ae350_c2_32_dsp.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c2_32_dsp.dts
rename to arch/riscv/boot/dts/andes/ae350_c2_32_dsp.dts
diff --git a/arch/riscv/boot/dts/ae350_c2_64.dts b/arch/riscv/boot/dts/andes/ae350_c2_64.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c2_64.dts
rename to arch/riscv/boot/dts/andes/ae350_c2_64.dts
diff --git a/arch/riscv/boot/dts/ae350_c2_64_d.dts b/arch/riscv/boot/dts/andes/ae350_c2_64_d.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c2_64_d.dts
rename to arch/riscv/boot/dts/andes/ae350_c2_64_d.dts
diff --git a/arch/riscv/boot/dts/ae350_c2_64_d_dsp.dts b/arch/riscv/boot/dts/andes/ae350_c2_64_d_dsp.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c2_64_d_dsp.dts
rename to arch/riscv/boot/dts/andes/ae350_c2_64_d_dsp.dts
diff --git a/arch/riscv/boot/dts/ae350_c2_64_dsp.dts b/arch/riscv/boot/dts/andes/ae350_c2_64_dsp.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c2_64_dsp.dts
rename to arch/riscv/boot/dts/andes/ae350_c2_64_dsp.dts
diff --git a/arch/riscv/boot/dts/ae350_c4_32.dts b/arch/riscv/boot/dts/andes/ae350_c4_32.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c4_32.dts
rename to arch/riscv/boot/dts/andes/ae350_c4_32.dts
diff --git a/arch/riscv/boot/dts/ae350_c4_32_d.dts b/arch/riscv/boot/dts/andes/ae350_c4_32_d.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c4_32_d.dts
rename to arch/riscv/boot/dts/andes/ae350_c4_32_d.dts
diff --git a/arch/riscv/boot/dts/ae350_c4_32_d_dsp.dts b/arch/riscv/boot/dts/andes/ae350_c4_32_d_dsp.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c4_32_d_dsp.dts
rename to arch/riscv/boot/dts/andes/ae350_c4_32_d_dsp.dts
diff --git a/arch/riscv/boot/dts/ae350_c4_32_dsp.dts b/arch/riscv/boot/dts/andes/ae350_c4_32_dsp.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c4_32_dsp.dts
rename to arch/riscv/boot/dts/andes/ae350_c4_32_dsp.dts
diff --git a/arch/riscv/boot/dts/ae350_c4_64.dts b/arch/riscv/boot/dts/andes/ae350_c4_64.dts
old mode 100755
new mode 100644
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c4_64.dts
rename to arch/riscv/boot/dts/andes/ae350_c4_64.dts
diff --git a/arch/riscv/boot/dts/ae350_c4_64_d.dts b/arch/riscv/boot/dts/andes/ae350_c4_64_d.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c4_64_d.dts
rename to arch/riscv/boot/dts/andes/ae350_c4_64_d.dts
diff --git a/arch/riscv/boot/dts/ae350_c4_64_d_dsp.dts b/arch/riscv/boot/dts/andes/ae350_c4_64_d_dsp.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c4_64_d_dsp.dts
rename to arch/riscv/boot/dts/andes/ae350_c4_64_d_dsp.dts
diff --git a/arch/riscv/boot/dts/ae350_c4_64_dsp.dts b/arch/riscv/boot/dts/andes/ae350_c4_64_dsp.dts
similarity index 100%
rename from arch/riscv/boot/dts/ae350_c4_64_dsp.dts
rename to arch/riscv/boot/dts/andes/ae350_c4_64_dsp.dts
diff --git a/arch/riscv/boot/dts/qemu_c1_32.dts b/arch/riscv/boot/dts/andes/qemu_c1_32.dts
similarity index 100%
rename from arch/riscv/boot/dts/qemu_c1_32.dts
rename to arch/riscv/boot/dts/andes/qemu_c1_32.dts
diff --git a/arch/riscv/boot/dts/qemu_c1_64.dts b/arch/riscv/boot/dts/andes/qemu_c1_64.dts
old mode 100755
new mode 100644
similarity index 100%
rename from arch/riscv/boot/dts/qemu_c1_64.dts
rename to arch/riscv/boot/dts/andes/qemu_c1_64.dts
diff --git a/arch/riscv/boot/dts/qemu_c2_32.dts b/arch/riscv/boot/dts/andes/qemu_c2_32.dts
similarity index 100%
rename from arch/riscv/boot/dts/qemu_c2_32.dts
rename to arch/riscv/boot/dts/andes/qemu_c2_32.dts
diff --git a/arch/riscv/boot/dts/qemu_c2_64.dts b/arch/riscv/boot/dts/andes/qemu_c2_64.dts
old mode 100755
new mode 100644
similarity index 100%
rename from arch/riscv/boot/dts/qemu_c2_64.dts
rename to arch/riscv/boot/dts/andes/qemu_c2_64.dts
diff --git a/arch/riscv/boot/dts/qemu_c4_32.dts b/arch/riscv/boot/dts/andes/qemu_c4_32.dts
similarity index 100%
rename from arch/riscv/boot/dts/qemu_c4_32.dts
rename to arch/riscv/boot/dts/andes/qemu_c4_32.dts
diff --git a/arch/riscv/boot/dts/qemu_c4_64.dts b/arch/riscv/boot/dts/andes/qemu_c4_64.dts
old mode 100755
new mode 100644
similarity index 100%
rename from arch/riscv/boot/dts/qemu_c4_64.dts
rename to arch/riscv/boot/dts/andes/qemu_c4_64.dts
-- 
2.34.1

